org.kde.spectacle.json

```diff
@@ -0,0 +1,75 @@
+{
+  "id": "org.kde.spectacle",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "spectacle",
+  "rename-icon": "spectacle",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=org.kde.kglobalaccel",
+    "--talk-name=org.kde.plasmashell"
+  ],
+  "modules": [
```

Comment:
Cleanup:

/include
/lib/cmake
/lib/systemd - seems to have a service, is this used or is there a build flag to turn it off?
*.a
/share/doc - if not needed
/share/man
/share/qlogging-categories5

Reply:
You can't turn https://invent.kde.org/graphics/spectacle/-/tree/master/dbus off.